src/fragments/lib/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -132,9 +132,9 @@ import flutter5 from "/src/fragments/lib/graphqlapi/flutter/advanced-workflows/3
 
 <Fragments fragments={{flutter: flutter5}} />
 
-## Combining Multiple Operations
+## Combining multiple GraphQL operations in a single request
```

Comment:
nit/question: Is there a recommendation in the styleguide for docs regarding capitalizing words in headers? The rest of the page seems to capitalize them although seems inconsistent throughout docs.